app/src/pages/resourceNotFound.tsx

```diff
@@ -0,0 +1,17 @@
+import {errors, links} from "../lang";
+import {FormattedText} from "@travelperksl/suitcase";
+import {Link} from "react-router-dom";
+
+const resourceNotFound = () => (
+    <div>
+        <h1><FormattedText size={'displayXL'}>404</FormattedText></h1>
+        <p><FormattedText size={'displayM'}>{errors.notFound.defaultMessage}</FormattedText></p>
+        <i>:( sadness</i>
```

Comment:
:sadge:

Reply:
^ More sad that github doesn't allow me to just copy emoju

Reply:
RFC to add `:sadge` to the Unicode emoji library